Makefile.in

```diff
@@ -9,7 +9,7 @@
 		> $@
 	$(_V) [ -z "$(UNIX2DOS)" ] || $(UNIX2DOS) $(UNIX2DOS_FLAGS) $@
 
-MD5SUM         ?= $(shell command -v md5sum 2>/dev/null)
+MD5SUM         ?= $(shell which md5sum 2>/dev/null)
```

Comment:
I thought the general trend in packaging was to remove `which` usages

Reply:
Hm, that could be. I cannot actually recall why I made this change in the first place, and the commit message does not mention it (which is unlike me), so I suspect I just accidentally put this change in this commit (which is also unlike me). This change is also not in the package I'm shipping in the Debian package, so it does not seem to be required there either.

In any case, I've force pushed an update to remove this change, leaving just the .sfo removal (and rebasing the PR at the same time).

Thanks!